### PR TITLE
Update validator bot configuration

### PR DESCRIPTION
**Pull request by juancebarberis**
## Description

Just a minor change on a Github issues template. 

## Changes

Title 'steps to reproduce' separated from 'Actual behavior & steps to reproduce'. It makes the github-bot validator fail if you don't put it separately.

## Test code and steps to reproduce

1. Create a new Github issue.
2. When sending it, the validator will not fail on steps to reproduce item.

## Checklist

- [ ] Included code example that can be used to test this change
- [ ] Updated TS types
- [ ] Added TS types tests
- [ ] Added unit / integration tests
- [x] Updated documentation
- [x] Ensured that CI passes

### CODE REVIEW

**Comment by jakub-gonet** [comment]
I think we should update the validator bot in that case. Especially that Reanimated v1 template suffers from the same problem.

**Comment by jakub-gonet** [comment]
Could you update [this file](https://github.com/software-mansion/react-native-reanimated/blob/a8d9720ee5312a43cc4e3beb02e70b2fc6a030ae/.github/workflows/issue-validator.yml) replacing Actual behavior with Actual behavior & steps to reproduce and removing Steps To Reproduce section?

Also, did you add a newline at the end of the file or you removed it?

**Comment by juancebarberis** [comment]
> Could you update this file replacing Actual behavior with Actual behavior & steps to reproduce and removing Steps To Reproduce section?

Done.

> Also, did you add a newline at the end of the file or you removed it?

Removed. Idk how that appears there.

File: .github/ISSUE_TEMPLATE/bug-report-v2.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ Tell us what's happening here.
 
 ### Expected behavior
 
-### Actual behavior & steps to reproduce
+### Actual behavior & steps to reproduce
```
Comment (jakub-gonet) [inline .github/ISSUE_TEMPLATE/bug-report-v2.md:22]:
```suggestion
### Actual behavior & steps to reproduce
```

**Review by jakub-gonet — APPROVED** [review]

**Pull request merged**